openverse_catalog/dags/common/storage/image.py

```diff
@@ -100,9 +100,14 @@ def add_item(
                              in this dictionary, and `license_url` is
                              given as an argument, the argument will
                              replace the one given in the dictionary.
-        raw_tags:            List of tags associated with the image
-        watermarked:         A boolean, or 't' or 'f' string; whether or
-                             not the image has a noticeable watermark.
+        raw_tags:            List of tags associated with the image.
+        category:            The image category. `None` will be replaced
+                             with the default category for provider. If you
+                             don't want to use the default, pass the string
+                             `UNKNOWN`, which will be replaced with None
+                             in the database. @see Flickr script for example usage.
```

Comment:
I'm not seeing any changes related to `UNKNOWN`, nor any existing logic for setting that to `NULL` in the database. Did I miss where this was occurring, or was it a change that was intended to be added?

Reply:
Oh, I first tried to implement it this way because the default category for Flickr was set to photograph. Then I realized that it's cleaner to just remove the Flickr default  category (and set some images to photograph while leaving others as None in its script) to unify the different provider behaviors. But I forgot to remove the comment!